sdk/python/feast/feature_store.py

```diff
@@ -153,6 +167,74 @@ def get_historical_features(
         )
         return job
 
+    def materialize(
+        self,
+        feature_views: Optional[List[str]],
+        start_date: datetime,
+        end_date: datetime,
+    ) -> None:
+        """
+        Materialize data from the offline store into the online store.
+
+        This method materializes feature data in the specified interval from either
```

Comment:
Let's try to use `loads` here maybe so that the user has a different verb instead of repeating `materialize` which they may not understand.